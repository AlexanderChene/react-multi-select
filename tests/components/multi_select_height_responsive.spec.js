import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import MultiSelectHeightResponsive from "../../src/components/multi_select_height_responsive";

describe("MultiSelectHeightResponsive", () => {
  test("without responsiveHeight", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<MultiSelectHeightResponsive />);
    expect(tree).toMatchSnapshot();
  });
  test("with responsiveHeight", () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
      <MultiSelectHeightResponsive responsiveHeight={"70%"} />
    );
    expect(tree).toMatchSnapshot();
  });
});
