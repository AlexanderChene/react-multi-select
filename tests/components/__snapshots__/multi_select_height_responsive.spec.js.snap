// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiSelectHeightResponsive with responsiveHeight 1`] = `
<ReactHeight
  dirty={true}
  getElementHeight={[Function]}
  hidden={false}
  onHeightReady={[Function]}
  style={
    Object {
      "height": "70%",
    }
  }
>
  <_class
    filterFunction={[Function]}
    height={400}
    items={Array []}
    responsiveHeight="70%"
    selectedItems={Array []}
  />
</ReactHeight>
`;

exports[`MultiSelectHeightResponsive without responsiveHeight 1`] = `
<ReactHeight
  dirty={true}
  getElementHeight={[Function]}
  hidden={false}
  onHeightReady={[Function]}
  style={
    Object {
      "height": "400px",
    }
  }
>
  <_class
    filterFunction={[Function]}
    height={400}
    items={Array []}
    responsiveHeight="400px"
    selectedItems={Array []}
  />
</ReactHeight>
`;
